### PR TITLE
Add release notes for c4b-azure 0.18.0

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

Add release notes for c4b-azure 0.18.0

## Motivation and Context

Publishing release notes for new release.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

**Review by Windos — APPROVED** [review]
LGTM, thanks @vexx32!

File: input/en-us/c4b-environments/azure/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,19 @@ Description: Release Notes for the Chocolatey for Business Azure Environment
 >
 > Issue links may not be publicly available at this time.
 
+## 0.18.1 (February 21, 2023)
```
Comment (Windos) [inline input/en-us/c4b-environments/azure/release-notes.md:14]:
```suggestion
## 0.18.0 (February 21, 2023)
```

Comment (Windos) [reply input/en-us/c4b-environments/azure/release-notes.md:14]: Oops, I missed one thing!

**Review by Windos — CHANGES_REQUESTED** [review]
I missed this, small change then can merge

File: input/en-us/c4b-environments/azure/release-notes.md (hunk 1 above)
Comment (vexx32) [reply input/en-us/c4b-environments/azure/release-notes.md]: How did I miss that, wow. Thanks for catching that 😂

**Comment by vexx32** [comment]
@Windos all fixed up, thanks for catching that one! 💖

**Review by Windos — APPROVED** [review]
Looks good to me!

**Pull request merged**